assets/src/application/services/theme/theme.ts

```diff
@@ -2,8 +2,11 @@ import { theme as chakraTheme, DefaultTheme } from '@chakra-ui/core';
 
 type CustomTheme = DefaultTheme;
 
+const zIndices = { ...chakraTheme.zIndices, popover: 1501 };
```

Comment:
Tweaked this, but no luck: 

![image](https://user-images.githubusercontent.com/1477453/79959774-330d8280-848d-11ea-8a88-4fe24552ccdf.png)

Reply:
Based on my finding, that `z-index=1` is overriding the theme value here: https://github.com/chakra-ui/chakra-ui/blob/master/packages/chakra-ui/src/Menu/index.js#L334
***
so, it won't help by adjuting values at the theme level, in this case